features/steps/steps.py

```diff
@@ -119,7 +197,12 @@ def when_i_retry_run_command(context, command, user_spec, exit_codes):
 
 @when("I run `{command}` {user_spec}")
 def when_i_run_command(
-    context, command, user_spec, verify_return=True, stdin=None
+    context,
+    command,
+    user_spec,
+    verify_return=True,
+    stdin=None,
+    instance_type="uaclient",
```

Comment:
I think `instance_name` might be a slightly better parameter name here but :shrug: